app/src/main/java/ca/pkay/rcloneexplorer/Fragments/FileExplorerFragment.java

```diff
@@ -582,7 +582,9 @@ public void onCreateOptionsMenu(Menu menu, MenuInflater inflater) {
             menu.findItem(R.id.action_sync).setVisible(false);
         }
 
-        menu.findItem(R.id.action_wrap_filenames).setChecked(true);
+        SharedPreferences sharedPreferences = PreferenceManager.getDefaultSharedPreferences(context);
+        Boolean isWrapFilenames = sharedPreferences.getBoolean(getString(R.string.pref_key_wrap_filenames), true);
```

Comment:
With your changes, logic does not have a connection to the options menu any more. Move reading the preference into an instance variable in [onCreate#234](https://github.com/x0b/rcx/blob/07f33467fde3a496873116b2c4e2e601dae57385/app/src/main/java/ca/pkay/rcloneexplorer/Fragments/FileExplorerFragment.java#L234); also avoids disk I/O.